test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/RegistrationControllerSpec.scala

```diff
@@ -44,19 +50,43 @@ class RegistrationControllerSpec extends ControllerSpec {
   }
 
   override protected def afterEach(): Unit = {
-    reset(registrationPage)
+    reset(registrationPage,
+          mockRegistrationConnector,
+          mockIncorpIdConnector,
+          mockSubscriptionsConnector
+    )
     super.afterEach()
   }
 
   "Registration Controller" should {
 
-    "return 200" when {
+    "return 200 when display page method is invoked" when {
+
+      "an UK Limited Company journeyID exists" in {
```

Comment:
Do we need to cover off Sole Trader as well? I don't know if there's actually any difference in the Journey ID per org type?

Reply:
Not really. But that is a good shout!
Actually I should update the test to say `"a GRS journeyID exists"`